jina/clients/request/helper.py

```diff
@@ -59,13 +58,9 @@ def _build_doc_from_content():
         return _build_doc_from_content()
 
 
-def _add_docs(req, batch, data_type, _kwargs):
+def _add_docs(req: DataRequest, batch, data_type: DataInputType, _kwargs):
     da = DocumentArray()
     for content in batch:
-        if isinstance(content, tuple) and len(content) == 2:
-            d, data_type = _new_doc_from_data(content[0], data_type, **_kwargs)
-            da.append(d)
-        else:
-            d, data_type = _new_doc_from_data(content, data_type, **_kwargs)
-            da.append(d)
+        d, data_type = _new_doc_from_data(content, data_type, **_kwargs)
```

Comment:
u are removing these `_kwargs` I think it is not needed to spend time on this

Reply:
I will try remove them and see if they still pass tests. But it is possible to access the _kwargs here.

The _kwargs I removed are in another function where it is created but never used

Reply:
It technically might be used here. But it seems to never need it in the tests.
https://github.com/jina-ai/jina/blob/master/jina/clients/request/helper.py#L37-L38

Reply:
I think it's a good idea to remove it. As described in https://github.com/jina-ai/jina/pull/5568#issuecomment-1369785922, it creates weird silent failures.